recipe/conda_build_config.yaml

```diff
@@ -526,5 +526,5 @@ zeromq:
 zlib:
   - 1.2
 zstd:
-  - 1.3.3  # [not aarch64]
-  - 1.3.7  # [aarch64]
+  - 1.3.3  # [not (aarch64 or ppc64le)]
+  - 1.3.7  # [aarch64 or ppc64le]
```

Comment:
Only 1.3.8 was built for ppc64le right?

Reply:
You are right. that needs to be tweaked.